src/utils/ddb/ddb_vos.c

```diff
@@ -2109,3 +2113,72 @@ dv_dev_replace(const char *db_path, uuid_t old_devid, uuid_t new_devid)
 	vos_self_fini();
 	return rc;
 }
+
+int
+dv_run_prov_mem(const char *db_path, const char *scm_mount, unsigned int scm_mount_size)
+{
+	int          rc;
+	bool         md_on_ssd;
+	bool         need_mount = true;
```

Comment:
Initialize `need_mount` as `false`; otherwise it may `umount` unexpectedly when cleanup for failure.